.jest/fastify.env.ts

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
```

Comment:
nitpick: i know you met this code there, but why do we even need this 🤔

Reply:
not anymore... it was the first option, more out of the source way to define test environments ... but it didnt work with fastify... types